src/components/FAQDropDown.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withStyles } from '@material-ui/core/styles'
+import ButtonBase from '@material-ui/core/ButtonBase'
+
+const styles = {
+  buttonBase: {
+    borderRadius: 2,
+  },
+  FAQDropDownText: {
+    fontWeight: 'normal',
+    transition: 'color 300ms ease-in',
+    cursor: 'pointer',
+  },
+  dropdownText: {},
+}
+
+class FAQDropDown extends React.Component {
```

Comment:
What made you use a class here? Not looking for a refactor, just curious what tripped you up.

Reply:
i was copying some bare bone structure from some legacy code and I was a little hesitant about using a functional component because I didn't fully understand how refs behave differently with functional components at the time.  I get it now so I'll refactor after I get the draft up before we release